sdcm/cluster.py

```diff
@@ -749,6 +777,9 @@ def get_backtraces(self):
         Verify the number of backtraces stored, report if new ones were found.
         """
         self.wait_ssh_up(verbose=False)
+        if self.distro == 'ubuntu14':
+            # ubuntu14 doesn't has coredumpctl, skip it.
```

Comment:
http://manpages.ubuntu.com/manpages/xenial/man1/coredumpctl.1.html

Reply:
It's for 16.04

`http://manpages.ubuntu.com/manpages/trusty/en/man1/coredumpctl.1.html` isn't available